types/chrome/index.d.ts

```diff
@@ -13887,334 +13887,231 @@ declare namespace chrome {
-             * @since Chrome 8
-             */
+            /** The new state of the window. The 'minimized', 'maximized', and 'fullscreen' states cannot be combined with 'left', 'top', 'width', or 'height'. */
+            state?: `${WindowState}` | undefined;
+            /** If `true`, brings the window to the front; cannot be combined with the state 'minimized'. If `false`, brings the next window in the z-order to the front; cannot be combined with the state 'fullscreen' or 'maximized'. */
             focused?: boolean | undefined;
-            /** Optional. The offset from the left edge of the screen to move the window to in pixels. This value is ignored for panels. */
+            /** The offset from the left edge of the screen to move the window to in pixels. This value is ignored for panels. */
             left?: number | undefined;
         }
 
-        export interface WindowEventFilter {
-            /**
-             * Conditions that the window's type being created must satisfy. By default it will satisfy ['app', 'normal', 'panel', 'popup'], with 'app' and 'panel' window types limited to the extension's own windows.
-             */
-            windowTypes: windowTypeEnum[];
-        }
-
-        export interface WindowIdEvent extends chrome.events.Event<(windowId: number) => void> {
-            addListener(
-                callback: (windowId: number) => void,
-                filters?: WindowEventFilter,
-            ): void;
-        }
-
-        export interface WindowReferenceEvent extends chrome.events.Event<(window: Window) => void> {
-            addListener(
-                callback: (window: Window) => void,
-                filters?: WindowEventFilter,
-            ): void;
-        }
-
         /**
          * Specifies what type of browser window to create.
          * 'panel' is deprecated and is available only to existing whitelisted extensions on Chrome OS.
          * @since Chrome 44
          */
-        export type createTypeEnum = "normal" | "popup" | "panel";
+        export enum CreateType {
+            /** Specifies the window as a standard window. */
+            NORMAL = "normal",
+            /** Specifies the window as a popup window. */
+            POPUP = "popup",
+            /** @deprecated Specifies the window as a panel. */
+            PANEL = "panel",
+        }
 
         /**
-         * The state of this browser window.
-         * In some circumstances a window may not be assigned a state property; for example, when querying closed windows from the sessions API.
+         * The state of this browser window. In some circumstances a window may not be assigned a `state` property; for example, when querying closed windows from the {@link sessions} API.
          * @since Chrome 44
          */
-        export type windowStateEnum = "normal" | "minimized" | "maximized" | "fullscreen" | "locked-fullscreen";
+        export enum WindowState {
```

Comment:
This enum doesn't seem to exist at runtime. Please revert this change (same for others) or provide docs/evidence that it's a concrete value.